infrastructure/db/factories/testDb.js

```diff
@@ -3,19 +3,14 @@ const { models, initDb } = require("..");
 
 const testDatabase = async () => {
   await initDb();
-  models.teammember.create({ emailAddress : faker.internet.email(), password : faker.internet.password() })
-    .then(user => {
-      models.board.create({label : 'FactoryBoard'}).then(board => {
-      //board.setTeammembers(user, {});
-      //user.addBoard(board, {});
-      //only use one of them. they do the same => add relation to the pivot table
-      models.group.create({ label : 'groupFactory', boardId: board.id })
-      .then(group => {
-          models.task.create({ label : 'taskFactory', status : 'unfinished',
-            metadata : {}, groupId: group.id, teammemberId: user.id, teammeberId: user.id })
-      });
-    });
-  });
+  const member = await models.teammember.create({ emailAddress : faker.internet.email(), password : faker.internet.password() });
```

Comment:
Amazing! Love it, see how the  `await` operator makes things much more clear? from a standpoint of 'i don't like the readability of this code' i'm shifted to 'whoa, it's pretty much straightforward'